pkg/tfgen/docs_test.go

```diff
@@ -835,38 +917,40 @@ subtitle 2 content
 	assert.Equal(t, expected, groupLines(strings.Split(input, "\n"), "## "))
 }
 
-func TestParseArgFromMarkdownLine(t *testing.T) {
-	//nolint:lll
-	tests := []struct {
-		input          string
-		expectedName   string
-		expectedDesc   string
-		expectedFound  bool
-		expectedIndent bool
-	}{
-		{"* `name` - (Required) A unique name to give the role.", "name", "A unique name to give the role.", true, false},
-		{"* `key_vault_key_id` - (Optional) The Key Vault key URI for CMK encryption. Changing this forces a new resource to be created.", "key_vault_key_id", "The Key Vault key URI for CMK encryption. Changing this forces a new resource to be created.", true, false},
-		{"* `urn` - The uniform resource name of the Droplet", "urn", "The uniform resource name of the Droplet", true, false},
-		{"* `name`- The name of the Droplet", "name", "The name of the Droplet", true, false},
-		{"* `jumbo_frame_capable` -Indicates whether jumbo frames (9001 MTU) are supported.", "jumbo_frame_capable", "Indicates whether jumbo frames (9001 MTU) are supported.", true, false},
-		{"* `ssl_support_method`: Specifies how you want CloudFront to serve HTTPS", "ssl_support_method", "Specifies how you want CloudFront to serve HTTPS", true, false},
-		{"* `principal_tags`: (Optional: []) - String to string map of variables.", "principal_tags", "String to string map of variables.", true, false},
-		{"  * `id` - The id of the property", "id", "The id of the property", true, true},
-		{"  * id - The id of the property", "", "", false, false},
-		//In rare cases, we may have a match where description is empty like the following, taken from https://github.com/hashicorp/terraform-provider-aws/blob/main/website/docs/r/spot_fleet_request.html.markdown
-		{"* `instance_pools_to_use_count` - (Optional; Default: 1)", "instance_pools_to_use_count", "", true, false},
-		{"", "", "", false, false},
-		{"Most of these arguments directly correspond to the", "", "", false, false},
-	}
-
-	for _, test := range tests {
-		parsedLine := parseArgFromMarkdownLine(test.input)
-		assert.Equal(t, test.expectedName, parsedLine.name)
-		assert.Equal(t, test.expectedDesc, parsedLine.desc)
-		assert.Equal(t, test.expectedFound, parsedLine.isFound)
-		assert.Equal(t, test.expectedIndent, parsedLine.isIndented)
-	}
-}
+//TODO: readjust this
```

Comment:
Is this still a TODO or was it left in by mistake?

Reply:
ha, no, that's a real TODO!